image_provisioner/playbooks/build_ami.yaml

```diff
@@ -0,0 +1,77 @@
+- hosts: localhost
+  connection: local
+  become: no
+  tasks:
+
+  - name: launch temporary instance
+    connection: local
+    become: no
+    ec2:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      assign_public_ip: yes
+      region: "{{ region }}"
+      key_name: "{{ key_name }}"
+      group_id: "{{ group_id }}"
+      instance_type: "{{ instance_type }}"
+      vpc_subnet_id: "{{ vpc_subnet_id }}"
+      image: "{{ base_image }}"
+      volumes:
+        - device_name: /dev/sdb
+          volume_type: io1
+          volume_size: 30
```

Comment:
We override it in SVT in our AMIs.   When doing a lot of builds for different apps, we use 80G.   But, I think 30G is a good default and if folks need more, they can override on instance creation.    Going too big is wasteful because you can't go lower when creating the instance.